code/workspaces/infrastructure-api/src/kubernetes/deploymentGenerator.js

```diff
@@ -97,7 +97,6 @@ function createRStudioDeployment({ deploymentName, volumeMount, type, version })
       image: img.image,
       connectImage: img.connectImage,
     },
-    rstudioConfigMapName: nameGenerator.rStudioConfigMap(deploymentName),
```

Comment:
Bit confused here, it seems like the configmap is no longer being used with the deployment?

Reply:
That's right - now the configmap is being used with the ingress.

Reply:
Ah thanks! I didn't know that ingress annotations could be resolved via a configmap. Would it just be easier to specify it as a string instead if that's how it's resolved and it's relatively simple?

Reply:
This annotation has to be resolved by a configmap, see https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/configmap/#proxy-set-headers